### PR TITLE
Creating a generic landing page to test routing, views

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * Added a new route for '' (which is Django's way of doing what we do in Ruby: '/' in our routes file)
 
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Any background context you want to provide?

Still quite new and unsure how to use and develop on Django, so this exercise was meant to help me understand how routing works.
 
### Issues: 
* Addresses # **No issues yet**
* Closes #
 
### Screenshots (if appropriate): 
*
 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: app/boneyard/urls.py [hunk 1]
```diff
@@ -15,7 +15,9 @@
 """
 from django.contrib import admin
 from django.urls import path
+from . import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.hello),
```
Comment (TravisGM92) [inline app/boneyard/urls.py:22]: This is important to note. I guess to create a new route, one has to use this syntax. Notice how on line 21 they do `path('admin/'...` it seems as though you'd have to type that into the browser to navigate to the admin page but it's actually `/admin`. For example: `http://localhost:8000/admin`

Comment (TravisGM92) [reply app/boneyard/urls.py:22]: Also, the `views.hello` part corresponds to the method defined in the `views.py` file.

File: app/boneyard/views.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
+def hello(request):
+    text = """<h1>Welcome to the app!</h1>"""
+    return HttpResponse(text)
```
Comment (TravisGM92) [inline app/boneyard/views.py:5]: Not exactly sure what HttpResponse is yet. Notice how I have raw HTML inside triple quote marks. We're not in Rails anymore       :(

Comment (TravisGM92) [reply app/boneyard/views.py:5]: I have a feeling we won't need to type raw HTML in triple quotes all the time. 🤷‍♂️

File: app/boneyard/urls.py (hunk 1 above)
Comment (judithpillado) [reply app/boneyard/urls.py:22]: Thank you for the comments! It's easier to follow along this way

File: app/boneyard/views.py (hunk 2 above)
Comment (judithpillado) [reply app/boneyard/views.py:5]: I guess maybe this is something we'll find out along the way! Overall looks great and easy to follow

**Review by judithpillado — APPROVED** [review]

**Pull request merged**